.github/workflows/ci.yml

```diff
@@ -252,6 +252,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-2019, windows-2022]
+        enable_cri_sandboxes: [ "", "sandboxed"]
```

Comment:
```suggestion
        enable_cri_sandboxes: [ "", "sbserver"]
```

Reply:
in github actions log, it read well, then I thought about the log out and having to explain what sandboxed means sbserver vs the current cri server .. sandbox/podsandbox/sandboxed ... google has a project called sandbox where they use an API called the Sandboxed API (SAPI)